parquet/src/file/encryption.rs

```diff
@@ -38,28 +38,41 @@ pub struct ParquetEncryptionKeyInfo {
     pub key: ParquetEncryptionKey,
 }
 
+/// Tells what mode (and also the key value(s)) a file is to be encrypted in (when writing) or is
+/// permitted to be encrypted in (when reading).
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum ParquetEncryptionMode {
+    /// Means the file is unencrypted
+    Unencrypted,
+    /// Means the file is footer-encrypted -- well, fully-encrypted.  The same key is used for all
+    /// the columns too, in this implementation.
+    FooterEncrypted(ParquetEncryptionKeyInfo),
```

Comment:
emm, if it is fully encrypted — maybe it should be named similarly? Cause now it's a bit contractionary, nope?

Reply:
and the comment could be less confusing too.

Reply:
I have changed the comment.